Description: Removed date from log file name, it confuses logrotate.
Author: Charlie Smotherman <cjsmo@cableone.net>  

Index: ampache-3.6-git408e713+dfsg/lib/log.lib.php
===================================================================
--- ampache-3.6-git408e713+dfsg.orig/lib/log.lib.php	2012-06-02 13:15:46.000000000 -0500
+++ ampache-3.6-git408e713+dfsg/lib/log.lib.php	2012-06-02 21:39:53.000000000 -0500
@@ -41,7 +41,7 @@
 	$log_name	= $log_name ? $log_name : 'ampache';
 	$username	= $username ? $username : 'ampache';
 
-	$log_filename	= Config::get('log_path') . "/$log_name.$log_day.log";
+	$log_filename	= Config::get('log_path') . "/$log_name.log";
 	$log_line	= "$log_time [$username] ($event_name) -> $event_description \n";
 
 	// Do the deed
